src/main/java/com/cloudbees/plugins/credentials/CredentialsProvider.java

```diff
@@ -560,7 +560,15 @@ public static <C extends Credentials> List<C> lookupCredentialsInItem(@NonNull C
         for (CredentialsProvider provider : all()) {
             if (provider.isEnabled(item) && provider.isApplicable(type)) {
                 try {
-                    for (C c: provider.getCredentialsInItem(type, item, authentication, domainRequirements)) {
+                    List<C> credentials = provider.getCredentialsInItem(type, item, authentication, domainRequirements);
+                    // also lookup credentials as SYSTEM if granted for this item
+                    if (authentication != ACL.SYSTEM2
+                            && (item.getACL().hasPermission2(authentication, CredentialsProvider.USE_ITEM)
+                            || item.getACL().hasPermission2(authentication, CredentialsProvider.USE_OWN))) {
```

Comment:
Sorry, I think I expressed myself incorrectly.
With only the `CredentialsProvider.USE_OWN` permission, you should not be able to retrieve credentials with the `SYSTEM` authentication. But it should be needed to retrieve the current user's credentials (with the user's authentication).

Meaning that the previous call on line 563 should check for this permission, similar to what is done in this [code](https://github.com/jenkinsci/credentials-plugin/blob/04f5ec1f3743da5c21b8a10b41aaffb614d6d61f/src/main/java/com/cloudbees/plugins/credentials/CredentialsParameterValue.java#L102-L110).